apps/quickbooks_online/utils.py

```
@@ -636,39 +725,81 @@
-                class_attributes.append({'attribute_type': 'CLASS', 'display_name': 'class', 'value': qbo_class['FullyQualifiedName'], 'destination_id': qbo_class['Id'], 'active': qbo_class['Active']})
-                if qbo_class['Id'] in active_existing_classes:
+                class_attributes.append(
+                    {
+                        'attribute_type': 'CLASS',
+                        'display_name': 'class',
+                        'value': qbo_class['FullyQualifiedName'],
+                        'destination_id': qbo_class['Id'],
+                        'active': qbo_class['Active']
+                    }
+                )
+                if qbo_class['Id'] in active_existing_classes and not is_sync_after_timestamp_enabled:
```

Pylint (3.3.7)

[error] Possibly using variable 'active_existing_classes' before assignment

(E0606)